### ISSUE TITLE
Connect Profiles: Ethernet-Web-Auth filter does not work 

### ISSUE THREAD

**Issue by lzammit** [inverse-inc/packetfence#6250]
conf/profiles.conf

````
[Wired-WebAuth]
locale=
sources=AD
advanced_filter=
filter=connection_type:Ethernet-Web-Auth

[ZamSecure]
locale=
sources=AD
advanced_filter=
filter=ssid:ZamSecure
autoregister=enabled

[ZamOpen]
locale=
sources=AD,local
advanced_filter=
filter=ssid:ZamOpen
provisioners=ios,windows

[Wired-Mac-Auth]
locale=
sources=AD,Twilio
advanced_filter=
filter=connection_type:Ethernet-NoEAP
````

````
       'Wired-WebAuth' => {
                               'billing_tiers' => [],
                               'filter_match_style' => 'any',
                               'preregistration' => 'disabled',
                               'unbound_dpsk' => 'disabled',
                               'sms_pin_retry_limit' => '0',
                               'locale' => [],
                               'vlan_pool_technique' => 'username_hash',
                               'always_use_redirecturl' => 'disabled',
                               'login_attempt_limit' => '0',
                               'template_paths' => [
                                                     '/usr/local/pf/html/captive-portal/profile-templates/Wired-WebAuth',
                                                     '/usr/local/pf/html/captive-portal/profile-templates/default',
                                                     '/usr/local/pf/html/captive-portal/templates'
                                                   ],
                               'guest_modes' => '',
                               'description' => '',
                               'network_logoff_popup' => 'disabled',
                               'reuse_dot1x_credentials' => '0',
                               'sources' => [
                                              'AD'
                                            ],
                               'access_registration_when_registered' => 'disabled',
                               'block_interval' => 600,
                               'advanced_filter' => '',
                               'provisioners' => [],
                               'dot1x_recompute_role_from_portal' => 'enabled',
                               'status' => 'enabled',
                               'dot1x_unset_on_unmatch' => 'disabled',
                               'unreg_on_acct_stop' => 'disabled',
                               'root_module' => 'default_policy',
                               'sms_request_limit' => '0',
                               'network_logoff' => 'disabled',
                               'dpsk' => 'disabled',
                               'filter' => [
                                             'connection_type:Ethernet-Web-Auth'
                                           ],
                               'mac_auth_recompute_role_from_portal' => 'disabled',
                               'autoregister' => 'disabled',
                               'logo' => '/common/packetfence-cp.png',
                               'redirecturl' => 'http://www.packetfence.org/',
                               'scans' => [],
                               'self_service' => ''
                             },
````
Logs:

````
Apr  1 15:35:18 cluster1 packetfence_httpd.aaa: httpd.aaa(27706) INFO: [mac:00:e0:4c:68:09:e3] handling radius autz request: from switch_ip => (192.168.126.253), connection_type => Ethernet-NoEAP,switch_mac => (5c:50:15:c2:eb:0b), mac => [00:e0:4c:68:09:e3], port => 10111, username => "00e04c6809e3" (pf::radius::authorize)
Apr  1 15:35:18 cluster1 packetfence_httpd.aaa: httpd.aaa(27706) INFO: [mac:00:e0:4c:68:09:e3] Instantiate profile Wired-Mac-Auth (pf::Connection::ProfileFactory::_from_profile)
Apr  1 15:35:18 cluster1 packetfence_httpd.aaa: httpd.aaa(27706) INFO: [mac:00:e0:4c:68:09:e3] is of status unreg; belongs into registration VLAN (pf::role::getRegistrationRole)
Apr  1 15:35:18 cluster1 packetfence_httpd.aaa: httpd.aaa(27706) INFO: [mac:00:e0:4c:68:09:e3] (192.168.126.253) Added role registration to the returned RADIUS Access-Accept (pf::Switch::returnRadiusAccessAccept)
Apr  1 15:35:18 cluster1 packetfence_httpd.aaa: httpd.aaa(27706) INFO: [mac:00:e0:4c:68:09:e3] Adding web authentication redirection to reply using role: 'registration' and URL: 'http://192.168.126.100/Cisco::Catalyst_2960/sidbe707b' (pf::Switch::Cisco::Catalyst_2960::returnRadiusAccessAccept)
Apr  1 15:35:19 cluster1 packetfence_httpd.aaa: httpd.aaa(27706) INFO: [mac:00:e0:4c:68:09:e3] Updating locationlog from accounting request (pf::api::handle_accounting_metadata)
Apr  1 15:35:23 cluster1 packetfence_httpd.portal: httpd.portal(8476) INFO: [mac:00:e0:4c:68:09:e3] External captive portal detected ! (captiveportal::PacketFence::Model::Portal::Session::_build_dispatcherSession)
Apr  1 15:35:23 cluster1 packetfence_httpd.portal: httpd.portal(8476) INFO: [mac:00:e0:4c:68:09:e3] Detected external portal client. Using the IP 192.168.126.203 address in it's session. (captiveportal::PacketFence::Model::Portal::Session::_build_clientIP)
Apr  1 15:35:23 cluster1 packetfence_httpd.portal: httpd.portal(8476) INFO: [mac:00:e0:4c:68:09:e3] Instantiate profile Wired-Mac-Auth (pf::Connection::ProfileFactory::_from_profile)
Apr  1 15:35:23 cluster1 packetfence_httpd.portal: httpd.portal(7835) INFO: [mac:00:e0:4c:68:09:e3] External captive portal detected ! (captiveportal::PacketFence::Model::Portal::Session::_build_dispatcherSession)
Apr  1 15:35:23 cluster1 packetfence_httpd.portal: httpd.portal(7835) INFO: [mac:00:e0:4c:68:09:e3] Detected external portal client. Using the IP 192.168.126.203 address in it's session. (captiveportal::PacketFence::Model::Portal::Session::_build_clientIP)
Apr  1 15:35:23 cluster1 packetfence_httpd.portal: httpd.portal(7835) INFO: [mac:00:e0:4c:68:09:e3] Instantiate profile Wired-Mac-Auth (pf::Connection::ProfileFactory::_from_profile)
Apr  1 15:35:24 cluster1 pfqueue: pfqueue(8440) WARN: [mac:00:e0:4c:68:09:e3] Unable to pull accounting history for device 00:e0:4c:68:09:e3. The history set doesn't exist yet. (pf::accounting_events_history::latest_mac_history)
````

**Comment by fdurand** [comment]
Same here https://github.com/inverse-inc/packetfence/issues/4816

**Comment by julsemaan** [comment]
Lets close this in favor of #4816 since we're not going to get that fixed in 10.3 anyway
